integrationtests/src/main/java/io/kroxylicious/proxy/testcluster/ContainerBasedKafkaCluster.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.testcluster;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.rnorth.ducttape.unreliables.Unreliables;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.lifecycle.Startable;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import io.kroxylicious.proxy.testcluster.ClusterConfig.KafkaEndpoints;
+import io.kroxylicious.proxy.testcluster.ClusterConfig.KafkaEndpoints.Endpoint;
+import lombok.SneakyThrows;
+
+/**
+ * Provides an easy way to launch a Kafka cluster with multiple brokers.
+ */
+public class ContainerBasedKafkaCluster implements Startable, Cluster {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ContainerBasedKafkaCluster.class);
+    public static final int KAFKA_PORT = 9093;
+    public static final int ZOOKEEPER_PORT = 2181;
+    private static final DockerImageName DEFAULT_KAFKA_IMAGE = DockerImageName.parse("bitnami/kafka:latest");
+    private static final DockerImageName DEFAULT_ZOOKEEPER_IMAGE = DockerImageName.parse("bitnami/zookeeper:latest");
```

Comment:
Why these? Why not the Strimzi test-container-images?

Reply:
My idea (possibly misguided) was to avoid dependencies on strimzi.   However, this will be moot, as I think we'll now take the native approach.